addresspoints/src/main/resources/logback.xml

```diff
@@ -1,23 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
+
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <!-- encoders are assigned the type
-             ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
         <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+            <pattern>[%d][%-5level][%thread] %msg%n</pattern>
         </encoder>
     </appender>
 
-    <appender name="logstash" class="net.logstash.logback.appender.LogstashTcpSocketAppender">
-        <!-- remoteHost and port are optional (default values shown) -->
-        <remoteHost>${LOGSTASH_HOST:-localhost}</remoteHost>
-        <port>${LOGSTASH_PORT:-9001}</port>
-
-        <!-- encoder is required -->
-        <encoder class="net.logstash.logback.encoder.LogstashEncoder" />
-    </appender>
-
-    <root level="INFO">
-        <appender-ref ref="logstash" />
+    <root level="DEBUG">
```

Comment:
@hkeeler I think level="INFO" should be the default, DEBUG produces to much output and will probably slow down running service in production